components/RegistrationForm.tsx

```diff
@@ -302,116 +204,79 @@ const RegistrationForm = ({
     return list;
   };
 
+  const mutation = useMutation({
+    mutationFn: async (newFormData: mergeFields) => {
+      const response = await fetch('/api/register2023', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(newFormData),
+      });
+      if (!response.ok) {
+        // fetch api doesn't handle errors. More info in the docs here:
+        // https://tinyurl.com/fetchNoError
+        setSubmitError(true);
```

Comment:
They _should_ be near the general error message anyways since it's located adjacent to the submit button. Would love if the form scrolled to the correct field with the error, but definitely more effort than we can afford at the moment. Future improvement for sure :D